components/ILIAS/IndividualAssessment/PRIVACY.md

```diff
@@ -0,0 +1,116 @@
+# Individual Assessment Privacy
+
+Disclaimer: This documentation does not warrant completeness or correctness. Please
+report any missing or wrong information using the [ILIAS issue tracker](https://mantis.ilias.de)
+or contribute a fix via [Pull Request](docs/development/contributing.md#pull-request-to-the-repositories).
+
+### General Information
+
+The Individual Assessment is intended as a performance and skill evaluation tool.
+The submitted data can shape a person's career. Therefore, some data is intentionally
+impossible to delete or change for most or all user roles. This makes it more likely
```

Comment:
You can slightly change to:
Therefore, certain data is intentionally made possible to delete or change for most or all user rules. This ensures that Individual Assessment records are more likely to be acctepted as proof in court.